openshift/deploy/database.yml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
```

Comment:
There is a quite different deploy config for vmaas database. E.g. there is used ConfigMap resource. Also you can see how are probes used there. Try to ask @jdobes for more info.

Reply:
Any reason to do it in a separate resource ? Any information apart from password will be static.